TreeTracker/UI/Views/AddTree/AddTreeViewModel.swift

```diff
@@ -32,17 +32,19 @@ class AddTreeViewModel {
     private let settingsService: SettingsService
     private let locationDataCapturer: LocationDataCapturing
     private let planter: Planter
+    private var note: String
     private let treeUUID: String
 
     init(
         locationProvider: LocationProvider,
         treeService: TreeService,
         settingsService: SettingsService,
         locationDataCapturer: LocationDataCapturing,
-        planter: Planter
+        planter: Planter,
+        note: String
```

Comment:
Not sure we need to initialise with a note as we won't have one at this point. Unless I'm missing something.

Reply:
I now have note being passed around as an optional and only being unwrapped that the very end when it is needed. saveTree method can take an optional note because the TreeCapture has it as an optional property.